src/providers/consensus/client.py

```diff
@@ -74,11 +80,11 @@ def get_block_root(self, state_id: Union[SlotNumber, BlockRoot, LiteralState]) -
     @lru_cache(maxsize=1)
     def get_block_header(self, state_id: Union[SlotNumber, BlockRoot]) -> BlockHeaderFullResponse:
         """Spec: https://ethereum.github.io/beacon-APIs/#/Beacon/getBlockHeader"""
-        data, meta_data = self._get(
+        data, meta_data = cast(Tuple[dict, dict], self._get(
```

Comment:
you probably may use native types like `cast(tuple[dict, dict])`
but why this cast is here at all?

Reply:
because of mypy issue https://github.com/python/mypy/issues/11692#issuecomment-994746538